lib/wallet/index.ts

```diff
@@ -12,9 +13,11 @@ export class LibraWallet {
   private lastChild = 1;
   private accounts: { [address: string]: Account } = {};
 
-  constructor(config: WalletConfig) {
-    this.config = config;
-    const seed = Seed.fromMnemonic(config.mnemonic.split(' '), config.salt);
+  constructor(config?: WalletConfig) {
+    this.config = config || {};
+    const mnemonic = (this.config.mnemonic) ? this.config.mnemonic : generateMnemonic(256);
```

Comment:
Can you explicitly check for if `this.config.mnemonic === undefined` here. So we can let users know when they pass in other falsy values and not generate unknowingly for them.

Reply:
Thanks for the review i will do that. I was wondering should i move the generateMnemonic method into Mnemonic.ts, so we can call the method directly if we want to without going through LibraWallet.

Reply:
@bonustrack That sounds like a good idea. Please go with this approach.